test/cpp/dist_autograd/test_dist_autograd.cpp

```diff
@@ -52,6 +52,7 @@ TEST_F(DistAutogradTest, TestSendFunctionInvalidInputs) {
   // grad.
   send_function->setGrads({in1, torch::autograd::Variable()});
   EXPECT_THROW(send_function->apply({}), c10::Error);
+  autogradContainer_->releaseContext(autogradContext->contextId());
```

Comment:
Thanks for fixing this @deepali-c! Can you instead override the `TearDown` method in `DistAutogradTest` to clean up the context? You can retrieve the current context from the container and then release it? This is much cleaner since the test framework ensures `TearDown` is always called, whereas here if something throws an exception before `releaseContext`, we might not release the context causing other tests to fail.

Reply:
sure.

Reply:
I have added a change, @pritamdamania87 please review.